manticore/__main__.py

```diff
@@ -9,6 +9,7 @@
 from threading import Timer
 from core.smtlib import Expression
 from manticore import Manticore
+from utils.mlogging import set_verbosity
```

Comment:
There is the logging built-in module.. should we change it to manticore_logging?